app.config.js

```diff
@@ -1,6 +0,0 @@
-
```

Comment:
Why did you delete this File?

Reply:
No i dont!
Removing this File Breaks our build !

Reply:
oh sorry. ok let me add it back